modules/cert-manager-acme-http01.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-http01_{context}"]
+= Configuring an ACME issuer to solve HTTP01 challenges
+
+You can use {cert-manager-operator} to set up an ACME issuer to solve HTTPS01 challenges.
+
+// TODO: Specify taht this example us using let's encrypt
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+// * User workload to expose
+// TODO: ^ from gdoc, what does this mean?
+
+.Procedure
+
+. Create an ACME cluster issuer.
+
+.. Create a YAML file that defines the `ClusterIssuer` object:
++
+.Example `acme-cluster-issuer.yaml` file
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod                                           <1>
+spec:
+  acme:
+    preferredChain: ""
+    privateKeySecretRef:
+      name: example-issuer-account-key-prod                        <2>
+    server: https://acme-staging-v02.api.letsencrypt.org/directory <3>
+    solvers:
+      - http01:
+          ingress:
+            class: openshift-default                               <4>
+----
+<1> TODO: Need description or know if they need to change.
+<2> TODO: Need description or know if they need to change.
+<3> TODO: Specify the URL to access the ACME server's `directory` endpoint.
+<4> TODO - Specify `openshift-default` to use the Ingress-to-Route controller (for what purpose).
+
+
+.. Create the `ClusterIssuer` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f acme-cluster-issuer.yaml
+----
+
+. Create an ingress to expose the service of the user workload.
+
+.. Create a YAML file that defines a `Namespace` object:
+
++
+.Example `namespace.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: my-ingress-namespace <1>
+----
+<1> Specify the namespace for the ingress.
+
+.. Create the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f namespace.yaml
+----
+
+.. Create a YAML file that defines the `Ingress` object:
++
+.Example `ingress.yaml` file
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: sample-ingress                                           <1>
+  namespace: my-ingress-namespace                                <2>
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod             <3>
```

Comment:
@thejasn since you had me change letsencrypt-prod to letsencrypt-staging above, I'm going to update this one as well. Let me know if that's not right

Reply:
@thejasn Can you confirm that that change was correct?